### ISSUE TITLE
Metadata not in search response

### ISSUE THREAD

**Issue by jlomen** [box/box-typescript-sdk-gen#339]
- [x] I have checked that the [SDK documentation][sdk-docs] doesn't solve my issue.
- [x] I have checked that the [API documentation][api-docs] doesn't solve my issue.
- [x] I have searched the [Box Developer Forums][dev-forums] and my issue isn't already reported (or if it has been reported, I have attached a link to it, for reference).
- [x] I have searched [Issues in this repo][github-repo] and my issue isn't already reported.

### Description of the Issue

We have been trying to migrate from the node sdk to this typescript sdk.
We use custom metadata for our files and are having issues getting the metadata in search responses when using this SDK
The apis we are using are
`client.search.searchForContent(params);`
and
`client.search.searchByMetadataQuery(params);`

No matter how many metadata fields we add to the `fields` parameter, we don't get any in the response
Our fields param looks like
```
fields: ['name', 'version_number', `metadata.enterprise_1234.fdl_template.dataEnvironment`]
```

We also notice the structure of the metadata response has changed in this SDK compared to the node sdk.
This is what we see in the response, notice the `dataEnvironment` is not present
```json
{
    name: "test_file.txt",
    metadata: {
        extraData: {
            enterprise_1234: {
                fdl_template: {
                    parent: 'file_997',
                    template: 'fdl_template',
                    scope: 'enterprise_1234',
                    version: 1
                    // *** NO METADATA HERE ***
                }
            }
        }
    }
}
```

### Steps to Reproduce

1. Use the search api's listed above
2. Search for files and use metadata in fields to get metadata in the response
3. Look at response
4. Expected metadata in not in response

### Expected Behavior

We expect to see the meta data in the response 
This was the behavior in the node sdk

### Versions Used

Typescript SDK: 1.5.1
Platform: Node, Typescript, Browser
Node.js version 20

[sdk-docs]: https://github.com/box/box-typescript-sdk-gen/tree/main/docs
[api-docs]: https://developer.box.com/docs
[dev-forums]: https://community.box.com/t5/Platform-and-Development-Forum/bd-p/DeveloperForum
[github-repo]: https://github.com/box/box-typescript-sdk-gen/search?type=Issues

**Comment by arjankowski** [comment]
Hello @jlomen,
Indeed, the SDK does not return these fields. This is due to an error in the schema that we use to generate our SDKs. 
We are already working on a fix, and we will let you know as soon as it's done.

Best regards,
Artur

**Comment by arjankowski** [comment]
Hi @jlomen,
I’m closing this issue because we have just released a new version **1.6.0** which includes a fix for the issue you mentioned. 

If you encounter any problems feel free to reopen it or create a new one.

Regards,
Artur
